### ISSUE TITLE
NTR: homologous chromosome segregation

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#15660]
NTR homologous chromosome separation
descendant of
homologous chromosome segregation

plus "regulation" terms

(note that "separation" is much narrower than segregation. Separation is about "kleisin cleavage" and release from cohesion, segregation covers spindle extension in anaphase, and preceding events of chromosome alignment, microtubule attachment etc  etc)

This arrangement will mirror the "mitotic sister chromatid segregation/separation" terms.

I also need  regulation terms (at leaset I need positive regulation for now, but here we know that negative regulation exists (via the APC), so this could be added too.

definition
The process in which sister chromatids are physically detached from each other during meiosis.

(this mirrors the mitotic def)

I think *both* of these definitions (mitotic and meiotic) could be extended so that people don't get confused between segregation and separation.

Add:
"this occurs when the mitotic/meiotic kleisin cohesin subunit is cleaved by separase/separin after the meiotic/mitotic spindle  checkpoint is satisfied.

delete mitotic/meiotic as appropriate.
(separase/separin are appropriate for both it's interchangeable names for the same protein).

**Comment by mah11** [comment]
> definition
> The process in which sister chromatids are physically detached from each other during meiosis.
> 
> (this mirrors the mitotic def)

This will not work for homologous chromosome separation - in meiosis I, homologs (i.e. whole replicated chromosomes) separate from each other but the two chromatids of each homolog stay together. Chromatids separate in meiosis II (and the term for that exists - GO:0051757 ! meiotic sister chromatid separation).

**Comment by ValWood** [comment]
Of course, I thought it was odd that I couldn't find it when searching!

**Comment by ValWood** [comment]
intead I need +ve  regulation for this existing term:

"meiotic sister chromatid arm separation"
